starknet_devnet/blueprints/rpc/structures/payloads.py

```diff
@@ -330,6 +336,13 @@ def rpc_declare_transaction(transaction: DeclareSpecificInfo) -> RpcDeclareTrans
         "nonce": rpc_felt(transaction.nonce),
         "type": rpc_txn_type(transaction.tx_type.name),
     }
+    if transaction.version == 2:
```

Comment:
Let's not use the hardcoded value here. We have constants for this in `starknet_devnet/constants.py`.